### DISCUSSION TITLE
Pinned discussion background image -- Allow custom images

### DISCUSSION THREAD

**Discussion by tallguyjenks** [community/community#2871] [Discussions] [answered]
on a community support repo like mine for answering and recording [FAQ](https://github.com/BryanJenksCommunity/FAQ/discussions) being able to upload a custom image for the pinned discussion banner would be a great way to further highlight  that I want users to click that banner first thing to learn how i want them to interact with the repo/discussions instead of default colors and relying on the text: `[[Start Here]]`

![image](https://user-images.githubusercontent.com/29872822/101570275-eb3f5b00-398a-11eb-8cf6-27c2da176296.png)

**Comment by becca** [comment, marked as answer]
I believe this might be a duplicate of the request over in https://github.com/github/feedback/discussions/2896

**Reply by tallguyjenks** [reply]
sure enough
